internetarchive/search.py

```diff
@@ -153,7 +153,7 @@ def _scrape(self):
             if j.get('error'):
                 yield j
             if not num_found:
-                num_found = int(j['total'])
+                num_found = int(j['total'] or 0)
             if not self._num_found:
                 self._num_found = num_found
```

Comment:
It would be awesome if we had an empty search test case that failed with the current code and passed with the proposed code.

The GitHub Actions tests have been updated so another push to the pull request should pass all GitHub Actions.

Please remove line 155 and then...
```suggestion
            num_found = num_found or int(j['total'] or 0)
            self._num_found = self._num_found or num_found
```